spec/views/reports/regions/_header.html.erb_spec.rb

```diff
@@ -0,0 +1,66 @@
+require "rails_helper"
+
+RSpec.describe "reports/regions/_header.html.erb", type: :view do
+  let(:organization) { create(:organization) }
+  let(:current_admin) { create(:admin, :manager) }
+  let(:organization_region) { create(:region, name: "Organization", region_type: "organization", path: "organization") }
+  let(:organization_region_1) { create(:region, name: "facility", region_type: "facility", path: "organization.state.district") }
```

Comment:
**suggestion**: it's better to call this `facility_region`, since this represents a facility, and not an organization.